resources/application-connector/charts/connector-service/templates/helm-certs-removal-backup.yaml

```diff
@@ -0,0 +1,98 @@
+{{- range $key, $value := .Values.certs -}}
+{{- $_ := set $ "key" $key }}
+{{- $_ := set $ "value" $value }}
+---
```

Comment:
This job and all its related resources should be restricted with disableLegacyConnectivity flag